operations/availabilityblocks.md

```diff
@@ -83,8 +83,6 @@ Returns all availability blocks filtered by services, unique identifiers and oth
 * `Confirmed` - The block deducts availability and can have reservations assigned.
 * `Optional` - The block deducts availability and cannot have reservations assigned.
 * `Inquired` - The block does not deduct availability and cannot have reservations assigned \(waitlist\).
-* `Released` - The block deducts availability, but only for reservations assigned to the block before release. Remaining availability is returned back to general availability \(waitlist\).
```

Comment:
Those items are listed here in purpose. It is a strict list of all possible states in the future. 
Unused items are marked - Waitlist. 
what are you trying to achive?
If it is confusing, let's discuss it with Mike, to make it clear. But removing something is not a solution :-)

+peter you can not easily change it to a dynamic list as this is a breaking change and partners would start receiving unexpected values in the future. 
Does it make sense to run the whole process?

Reply:
My suggestions as follows, happy to receive input on this:

1. Keep the list of possible states
2. If some states are not yet supported, we can say so, but I don't like the word "waitlist" as for me that's not clear what it means, better to say "not yet supported"
3. Hopefully if a client calls 'Get all availability blocks' with any non-supported states in the request, they get a sensible error message back
4. As an aside, I recommend updating the note against each operation to say "Restricted!" and do the same on the Operations home page, like we do for 'Add payment command'
5. As a further aside, I would also consider updating the descriptions for the different states to be a bit clearer - see below

    * **Confirmed** // fully confirmed for use; removed from general availability; reservations can be assigned to this block
    * **Optional** // removed from general availability but not fully confirmed; reservations cannot yet be assigned to this block
    * **Inquired** // inactive, provisional inquiry only; _not_ removed from general availability at this stage; no reservations can be assigned to this block
    * **Released** // the block has been released back into general availability; where reservations have already been assigned to this block, they are not released back into general availability
    * **Canceled** // inactive, canceled; _not_ removed from general availability at this stage; no reservations can be assigned to this block